### PR TITLE
Add test scenario 10

### PR DESCRIPTION
**Pull request by Mr-Leshiy**

### CODE REVIEW

File: src/blockchain/alt_block_addon.cpp [hunk 1]
```diff
@@ -90,10 +90,22 @@ std::vector<typename ATV::id_t>& AltBlockAddon::getPayloadIdsInner<ATV>() {
...
+    res += atv.toPrettyString() + ", ";
+  }
+  res += "VTBs: ";
+  for (const auto& vtb : _vtbids) {
+    res += vtb.toPrettyString() + ", ";
+  }
+  for (const auto& vbk : _vbkblockids) {
+    res += vbk.toPrettyString() + ", ";
+  }
+  return res;
```
Comment (Warchant) [inline src/blockchain/alt_block_addon.cpp:108]: maybe try to use fmt::join to join all ids?

File: include/veriblock/blob.hpp [hunk 2]
```diff
@@ -269,4 +271,24 @@ struct std::hash<altintegration::Blob<N>> {
...
+    }
+
+    // Return an iterator past the end of the parsed range:
+    return it;
+  }
+
+  template <typename FormatContext>
+  auto format(const altintegration::Blob<N>& val, FormatContext& ctx)
+      -> decltype(ctx.out()) {
+    return format_to(ctx.out(), "{:s}", val.toPrettyString());
```
Comment (Warchant) [inline include/veriblock/blob.hpp:290]:
```suggestion
    return format_to(ctx.out(), "{:s}", val.toHex());
```

File: src/blockchain/alt_block_addon.cpp [hunk 3]
```diff
@@ -90,10 +90,14 @@ std::vector<typename ATV::id_t>& AltBlockAddon::getPayloadIdsInner<ATV>() {
 }
 
 std::string AltBlockAddon::toPrettyString() const {
-  return fmt::sprintf("ATV=%d, VTB=%d, VBK=%d",
-                      _atvids.size(),
-                      _vtbids.size(),
-                      _vbkblockids.size());
+  return fmt::format(
+      "ATV={}, VTB={}, VBK={} \n ATVs:[{}] \n VTBs:[{}] \n VBKs:[{}]",
```
Comment (Warchant) [inline src/blockchain/alt_block_addon.cpp:94]:
```suggestion
      "ATV={}[{}], VTB={}[{}], VBK={}[{}]",
```

File: src/blockchain/vbk_block_addon.cpp [hunk 4]
```diff
@@ -66,7 +66,8 @@ void VbkBlockAddon::setNull() {
 }
 
 std::string VbkBlockAddon::toPrettyString() const {
-  return fmt::sprintf("VTB=%d", _vtbids.size());
+  return fmt::format(
+      "VTB={} \n VTBs:[{}]", _vtbids.size(), fmt::join(_vtbids, ","));
```
Comment (Warchant) [inline src/blockchain/vbk_block_addon.cpp:70]:
```suggestion
      "VTB={}[{}]", _vtbids.size(), fmt::join(_vtbids, ","));
```

File: src/blockchain/alt_block_addon.cpp (hunk 1 above)
Comment (Mr-Leshiy) [reply src/blockchain/alt_block_addon.cpp]: fixed

**Review by Warchant — APPROVED** [review]

**Pull request merged**